packages/cli/src/tools/config/samples.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * @flow
+ *
+ * Sample configuration objects that are used for `jest-validate`.
+ *
+ * See `./types.flow` for explanation of what
+ * the properties are responsible for.
+ */
+import {multipleValidOptions} from 'jest-validate';
+
+import {type UserConfigT, type LegacyDependencyUserConfigT} from './types.flow';
+
+const dependencyConfig = {
+  dependency: {
+    platforms: {
+      android: {
+        sourceDir: './android/app',
+        manifestPath: './path/to/AndroidManifest.xml',
+        packageImportPath: 'import com.my.package.MyReactPackage',
+        packageInstance: 'new MyReactPackage()',
+      },
+      ios: {
+        project: 'MyProject.xcodeproj',
+        sharedLibraries: ['libz'],
+        libraryFolder: 'Libraries',
+      },
+    },
+    assets: ['./path/to/asset.png'],
+    hooks: {
+      prelink: './path/to/a/command.sh',
+      postlink: './path/to/a/command.sh',
+    },
+    params: [
+      {
+        type: 'input',
+        name: 'myKey',
+        message: 'What is your deployment key value?',
+      },
+    ],
+  },
+  platforms: {
+    windows: './platform.js',
+  },
+  commands: ['./local-cli/runWindows/runWindows.js'],
+};
+
+const projectConfig = {
+  dependencies: {
+    'react-native-webview': {
```

Comment:
@thymikee @cpojer, is there a way in `jest-validate` to tell it it's a map with keys? Right now, it fails because when I define a different key than "react-native-webview", it obviously doesn't match.

Is the only solution to split it into two validate calls, one for `projectConfig` and then, for each `dependencies[X]`?